web/docs/guides/auth/auth-bitbucket.mdx

```diff
@@ -0,0 +1,87 @@
+---
+id: auth-bitbucket
+title: "Setting up Bitbucket Auth"
+description: Add Bitbucket OAuth to your Supabase project
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Setting up Bitbucket logins for your application consists of 3 parts:
```

Comment:
Trim wordiness wherever possible. eg:

Bitbucket OAuth consists of three broad steps.

is more consise than 

Setting up Bitbucket logins for your application consists of 3 parts.

Alternatively: `There are 3 steps required to set up Bitbucket logins:"